src/ui/veles_mainwindow.cc

```diff
@@ -647,6 +702,21 @@ QTabBar* MainWindowWithDetachableDockWidgets::dockWidgetToTab(
   return nullptr;
 }
 
+void MainWindowWithDetachableDockWidgets::splitDockWidget2(
+    QDockWidget* first, QDockWidget* second, Qt::Orientation orientation) {
+#ifdef SPLIT_DOCK_WIDGET_ACTUALLY_WORKS_AS_ONE_COULD_EXPECT
```

Comment:
This isn't defined anywhere and I don' think this is some built in define on any of our platforms.